WeakAuras/RegionTypes/Model.lua

```diff
@@ -1,5 +1,6 @@
 local SharedMedia = LibStub("LibSharedMedia-3.0");
 local L = WeakAuras.L;
+if WeakAuras.IsClassic then return end -- Models disabled for classic
```

Comment:
We can simply not ship models.lua for classic, instead of returning instantly. @Stanzilla what do you think?

Reply:
I agree, but we need to keep that line to keep dev build working for both version without .toc filtered by packager

Reply:
Ah, fair point. In that case we can add some #retail tags to to not attempt to load the file for the classic package, and leave this bit in so that the dev build works properly.